samples/snippets/sample_default_values.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A sample script showing how to handle default values when communicating
+with the Compute Engine API.
+"""
+# [START compute_instances_verify_default_value]
+# [START compute_usage_report_set]
+# [START compute_usage_report_get]
+# [START compute_usage_report_disable]
+from google.cloud import compute_v1
+# [END compute_usage_report_disable]
+# [END compute_usage_report_get]
+# [END compute_usage_report_set]
+
+
+# [START compute_usage_report_set]
+def set_usage_export_bucket(project_id: str, bucket_name: str,
+                            report_name_prefix: str = "") -> None:
+    """
+    Set Compute Engine usage export bucket for the Cloud Project.
+    This sample presents how to interpret default value for the name prefix
+    parameter.
+
+    Params:
+        project_id: project ID or project number of the project to update.
+        bucket_name: Google Cloud Storage Bucket used to store Compute Engine
+            usage reports. An existing Google Cloud Storage bucket is required.
+        report_name_prefix: Report Name Prefix which defaults to an empty string
+            to showcase default values behaviour.
+    """
+    usage_export_location = compute_v1.UsageExportLocation({
+        'bucket_name': bucket_name,
+        'report_name_prefix': report_name_prefix
+    })
+
+    if not report_name_prefix:
+        # Sending empty value for report_name_prefix, will result with the
+        # next usage report generated will have the default prefix value
```

Comment:
Please consider changing to (it seems more grammatically correct to me):
"Sending empty value for report_name_prefix, will result the next usage report generated to have the default prefix value "usage_gce"."

Reply:
OK